### DISCUSSION TITLE
Archlinux32 also works!

### DISCUSSION THREAD

**Discussion by ghost** [copy/v86#842] [General]
Tested ISO: https://mirror.math.princeton.edu/pub/archlinux32/archisos/archlinux32-2022.12.01-i486.iso
Download snapshot: https://files.catbox.moe/99vx4g.bin (requires 1024 MB of RAM)

Takes about 4 minutes to boot.
Unfortunately the OS is too bloated to even press a key when logged in. 

Archlinux32 is better since it is more modern base.

![Untitled](https://user-images.githubusercontent.com/111705094/228975749-58459faf-8a8a-45f7-9973-07d93127018e.png)

**Comment by spetterman66** [comment]
This is the "Arch Linux" profile on the V86 website, no setup required there :)
